modules/rostests/apitests/user32/SW_FORCEMINIMIZE.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     Test for ShowWindow.WS_FORCEMINIMIZE
```

Comment:
(SW_FORCEMINIMIZE) and as mentioned above, it may make more sense to move these tests into the ShowWindow.c file.

Reply:
Moved to `ShowWindow.c`.